cmd/crates/soroban-spec-typescript/ts-tests/package.json

```diff
@@ -1,34 +1,31 @@
 {
-	"private": true,
-	"type": "module",
-	"scripts": {
-		"postinstall": "npm run deploy && npm run bindings || { echo \"Make sure you're running standalone RPC network on localhost:8000\" && exit 1; }",
-		"fund": "./soroban config identity fund",
-		"bindings:custom-types": "./soroban contract bindings typescript --contract-id $(cat contract-id-custom-types.txt) --output-dir ./node_modules/test-custom-types --overwrite",
-		"bindings:hello-world": "./soroban contract bindings typescript --contract-id $(cat contract-id-hello-world.txt) --output-dir ./node_modules/test-hello-world --overwrite",
-		"bindings": "npm run bindings:custom-types && npm run bindings:hello-world",
-		"deploy:custom-types": "(./soroban contract deploy --wasm ../../../../target/wasm32-unknown-unknown/test-wasms/test_custom_types.wasm) > contract-id-custom-types.txt",
-		"deploy:hello-world": "(./soroban contract deploy --wasm ../../../../target/wasm32-unknown-unknown/test-wasms/test_hello_world.wasm) > contract-id-hello-world.txt",
-		"deploy": "npm run deploy:custom-types && npm run deploy:hello-world",
-		"test": "ava"
-	},
-	"devDependencies": {
-		"@ava/typescript": "^4.1.0",
-		"@types/node": "^20.4.9",
-		"ava": "^5.3.1",
-		"soroban-client": "1.0.0-beta.2",
-		"typescript": "^5.1.6"
-	},
-	"ava": {
-		"typescript": {
-			"rewritePaths": {
-				"src/": "build/"
-			},
-			"compile": "tsc"
-		}
-	},
-  "engines": {
-    "node": ">=20.0.0",
-    "npm": ">=9.8.1"
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "lint": "eslint src/*",
+    "postinstall": "./initialize.sh",
+    "test": "npm run lint && ava"
+  },
+  "devDependencies": {
+    "@ava/typescript": "^4.1.0",
+    "@types/node": "^20.4.9",
+    "@typescript-eslint/eslint-plugin": "^6.10.0",
+    "@typescript-eslint/parser": "^6.10.0",
+    "ava": "^5.3.1",
+    "dotenv": "^16.3.1",
+    "eslint": "^8.53.0",
+    "soroban-client": "1.0.0-beta.2",
```

Comment:
another PR later to update this to [beta.4](https://github.com/stellar/js-soroban-client/releases/tag/v1.0.0-beta.4)? and/or to stellar-sdk

Reply:
I think it makes sense to update now. Is stellar-sdk ready and preferred to beta.4?

Reply:
yes and yes :+1: